app/DTO/ImageResizeDTO.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace app\DTO;
+
+class ImageResizeDTO
+{
+
+    private $width;
+    private $height;
+    private $path;
+    private $image;
+    private $filename;
+    private $callback;
+    private $folder;
+
+    public function getWidth()
+    {
+        return $this->width;
+    }
+
+    public function setWidth($width)
+    {
+        $this->width = $width;
+        return $this;
```

Comment:
Here and everywhere if `return` statements is not alone, there should be 1 blank line above.